sdk/textanalytics/Azure.AI.TextAnalytics/src/Internal/MicrosoftCognitiveLanguageServiceRestClient.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.TextAnalytics.Models;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    internal partial class MicrosoftCognitiveLanguageServiceRestClient
```

Comment:
That doesn't look nearly as scary as I would have thought.

Reply:
I just don't like to do manual things when our tooling can do it for us. The less we have to think and change, the better :)